apple-nav/src/components/SubNav/SubNav.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import styles from "./SubNav.module.scss";
+import logo from "../../images/mac/iMac.png";
+import { Link } from "react-router-dom";
+
+export default class SubNav extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      mac: [
+        "MacBook",
+        "MacBook Air",
+        "MacBook Pro",
+        "iMac",
+        "iMac Pro",
+        "Mac Pro",
+        "Mac Mini",
+        "Accessories",
+        "Mojave",
+        "Compare"
+      ],
+      ipad: "",
+      iphone: "",
+      watch: "",
+      tv: "",
+      music: ""
+    };
+  }
+  render() {
+    const section = this.props.match.params.section;
+
+    const subsections = this.state[section];
+
+    let links = [
+      <div className={styles.link}>
+        <p>
+          I was lazy and only made the content for Mac okay? The coding
```

Comment:
lol yes, that is fine. I don't see a point in doing busy work for the sake of busy work unless it was for a major portfolio piece. So this is fine